peril-settings.json

```diff
@@ -22,7 +22,8 @@
                 "Automattic/peril-settings@org/android.ts",
                 "Automattic/peril-settings@org/github/label.ts",
                 "Automattic/peril-settings@org/github/milestone.ts",
-                "Automattic/peril-settings@org/github/diff-size.ts"
+                "Automattic/peril-settings@org/github/diff-size.ts",
+                "Automattic/peril-settings@org/github/pr-for-subtree.ts"
```

Comment:
Since this is currently specific to the login subtree, could we rename this to something like `check-login-subtree.ts`?

Reply:
That's a great idea!  Done.